clusterloader/run-e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+cd e2e/ && go test -c -o e2e.test
+./e2e.test --host="127.0.0.1:8080" --ginkgo.v=true --ginkgo.focus="Cluster Loader" --kubeconfig="${HOME}/.kube/config" --viper-config=../config/test
```

Comment:
I would try to replicate the logic from:
https://github.com/kubernetes/kubernetes/blob/master/hack/ginkgo-e2e.sh

It seems ${KUBE_MASTER_URL} should do the job.

BTW - there might be still some missing flags here, but I guess we will figure it out once this is submitted and we will look into logs from jenkins runs - I don't know how to easily do it differently.